public/js/base/gallery.js

```diff
@@ -1,30 +1,29 @@
-define(['jquery', 'nunjucks', 'base/ui', 'moment', 'makeapi', 'localized', 'masonry', 'base/login'],
+define(["jquery", "nunjucks", "base/ui", "moment", "makeapi", "localized", "masonry", "base/login"],
```

Comment:
Let's pick either ' or " instead of having some files that are all-' and some files that are all-"